system_tests/batch_poster_test.go

```diff
@@ -909,3 +909,49 @@ func TestBatchPosterActuallyPostsBlobsToL1(t *testing.T) {
 		require.Len(t, restoredBlobs, 1)
 	}
 }
+
+func TestBatchPosterPostsReportOnlyBatchAfterMaxEmptyBatchDelay(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	builder := NewNodeBuilder(ctx).
+		DefaultConfig(t, true).
+		TakeOwnership()
+
+	builder.nodeConfig.DelayedSequencer.Enable = true
+	builder.nodeConfig.DelayedSequencer.FinalizeDistance = 1
+	builder.nodeConfig.BatchPoster.MaxEmptyBatchDelay = time.Second
+	builder.nodeConfig.BatchPoster.PollInterval = time.Hour
+	builder.nodeConfig.BatchPoster.MaxDelay = 0
+
+	cleanup := builder.Build(t)
+	defer cleanup()
+
+	initialBatchCount := GetBatchCount(t, builder)
+
+	// Force immediate post of first batch.
+	builder.L2.ConsensusConfigFetcher.Set(builder.nodeConfig)
```

Comment:
I think this is redundant with https://github.com/OffchainLabs/nitro/blob/master/system_tests/common_test.go#L685-L690